### PR TITLE
Move Loris to its own ECS cluster

### PR DESCRIPTION
**Pull request by alicefuzier**
### What is this PR trying to achieve?
What the title says so that we can provision the ASG in different ways for Loris and for the API. This will allow us to add EBS volumes for #782 without having to add EBS volumes for API instances as well
### Who is this change for?
Devs

- [x] Run `terraform apply`.

### CODE REVIEW

**Comment by alicefuzier** [comment]
I think Loris can stay in the same VPC as the API and can continue to share the same ALB, but I'm not too sure about that, so let me know what you guys think

File: loris/terraform/asg.tf [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+module "loris_cluster_asg" {
+  source                = "../../terraform/ecs_asg"
+  asg_name              = "loris-cluster"
+  subnet_list           = ["${module.vpc_api.subnets}"]
```
Comment (alexwlchan) [inline loris/terraform/asg.tf]: You’ll need to pull this in from the shared stack.

File: loris/terraform/asg.tf [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  key_name              = "${var.key_name}"
+  instance_profile_name = "${module.ecs_loris_iam.instance_profile_name}"
```
Comment (alexwlchan) [inline loris/terraform/asg.tf]: This needs to be copied across.

**Comment by alexwlchan** [comment]
Resolves #955.

File: loris/terraform/ecr.tf [hunk 3]
```diff
@@ -1,9 +1,9 @@
 module "ecr_nginx_loris" {
-  source = "../terraform/ecr"
+  source = "../../terraform/ecr"
```
Comment (alexwlchan) [inline loris/terraform/ecr.tf]: Mildly surprised that this has changed, has something in the Makefile changed?

**Pull request merged**